minimal/docker-compose.yml

```diff
@@ -41,21 +39,24 @@ services:
     build: ./compose/nginx/production
     depends_on:
       - {{ project_name }}_web
-    volumes:
-      - ./media:/code/media
-      - ./staticfiles:/code/staticfiles
+    volumes_from:
+      - {{ project_name }}_web
```

Comment:
@pslobo for security reasons i wanted to avoid allowing nginx to have full `{{ project_name }}_web` access thats why i filtered the access to media and static. do you really need this?